docs/rum_getting_started.md

```diff
@@ -15,20 +15,26 @@ The Datadog Android SDK supports Android 4.4 (API level 19)+.
 
 ### Declare SDK as dependency
 
-Declare [dd-sdk-android][1] and the [gradle plugin][12] as a dependency in your `build.gradle` file.
+Declare [dd-sdk-android][1] and the [gradle plugin][12] as a dependency in your `build.gradle` file within your `app`directory
```

Comment:
> file within your `app` directory
The "application" module is usually named `app`, but it's not always the case, and a project can have multiple applications.
```suggestion
Declare [dd-sdk-android][1] and the [gradle plugin][12] as a dependency in your application module's `build.gradle` file
```